spec/fixtures/proofing/aamva/requests/verification_request.xml

```diff
@@ -1,24 +1,20 @@
-<soap:Envelope xmlns:aa='http://aamva.org/niem/extensions/1.0' xmlns:dldv='http://aamva.org/dldv/wsdl/2.1' xmlns:nc='http://niem.gov/niem/niem-core/2.0' xmlns:soap='http://www.w3.org/2003/05/soap-envelope'>
+<soap:Envelope xmlns:soap='http://www.w3.org/2003/05/soap-envelope' xmlns:dldv='http://aamva.org/dldv/wsdl/2.1' xmlns:aa='http://aamva.org/niem/extensions/1.0' xmlns:nc='http://niem.gov/niem/niem-core/2.0'>
```

Comment:
These keys were previously sorted alphabetically. `Formatters::Pretty` put them in this order and I do not understand the logic.